db/migrate/20220128222402_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
```

Comment:
Please see the below error which stopped the migrations to run. Kindly make sure all the migrations are running successfully.
![error1](https://user-images.githubusercontent.com/35504149/151596619-ed3ae68e-4e47-41ca-8199-ccf4c7332933.png)